### PR TITLE
add 4 lines to new.txt edit pr title

### PR DESCRIPTION
**Pull request by nadimalabbas**
pr body edit pr body

### CODE REVIEW

**Review by nadtesttest — CHANGES_REQUESTED** [review]
first request of changes

**Comment by nadimalabbas** [comment]
author comment

File: new.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+A new line for PR
+A 2nd line
```
Comment (nadtesttest) [inline new.txt:2]: add a comma

**Review by nadtesttest — CHANGES_REQUESTED** [review]
I requested a comma on line 2

File: new.txt (hunk 1 above)
Comment (nadimalabbas) [reply new.txt:2]: a comment on the review b y the author

Comment (nadimalabbas) [reply new.txt:2]: a comment by the author

Comment (nadimalabbas) [reply new.txt:2]: comment by the author

Comment (nadimalabbas) [reply new.txt:2]: a comment by the author

**Pull request merged**